src/sddp.jl

```diff
@@ -217,6 +217,17 @@ function set_custom_hook(
         return nothing
     end
 
+    function bellman_term_hook(model::JuMP.Model)
+        # This is the hook to be called at market clearing to make a tiebreaker weight
+        # to the fcf.
+        if use_fcf_in_clearing(inputs) || clearing_representation_must_read_cuts(inputs, run_time_options)
+            node = SDDP.get_node(model)
+            bellman_term = SDDP.bellman_term(node.bellman_function)
+            JuMP.set_objective_coefficient(model, bellman_term, market_clearing_tiebreaker_weight(inputs))
+        end
```

Comment:
@rafabench se for clearing cost based ai não precisa dessa multiplicação, certo? Acho que tem que mudar isso, só tem o ajuste no Hybrid ou no bid based